### DISCUSSION TITLE
Plex 2FA

### DISCUSSION THREAD

**Discussion by Tiwas** [Taxel/PlexTraktSync#1429] [Q&A]
Hi,
I just started using PlexTraktSync and had to disable 2FA on Plex to get it to log in. Should it be ok to enable 2FA again, or do I need to set something in the config?

**Comment by simonc56** [comment]
You should concatenate 2FA code to password (password2FACODE) at login as said in README, what is the error you get ?

**Reply by Tiwas** [reply]
Not sure if I did it correct, but if my password was "p4ssWord" and the 2FA from authenticator was "111333" I wrote "p4ssWord111333" in the password field. Got something about too many failed attempts. Disabling 2FA and only using the password worked.

**Comment by glensc** [comment]
outdated. current plextraktsync asks for 2fa separately
